### PR TITLE
Redraw, optimize and minify SVG and PNG assets

### PR DESCRIPTION
**Pull request by indiscipline**
The crown is redrawn:
* Now is symmetrical and centred on the canvas/viewbox
* The top arc of the top part of the crown is now a continuous curve of an ellipse
* All points are manually moved to 0.1pt precision (mostly to 0.5)
* PNG is now 8-bit and compressed

Asset size comparison:
| Asset             | Old   | New   | O/N %   |
|-------------------|-------|-------|-----|
| logo-crown.png    | 45121 | 10033 | 22% |
| logo-crown.svg    | 6678  | **813**   | **12%** |
| logo-design.svg   | 18294 | 4143  | 23% |
| logo-on-black.svg | 7086  | 1203  | 17% |

Old/New shape comparison:
logo-crown-comparison

> [!WARNING]  
> The potential backward-incompatibilities:
>  1. The width of the PNG file changed from 1113px to 1124px due to now being symmetrical and requiring a bit more space for the right half of the picture. The worst that can happen down the line if the asset is silently updated is a 11px trim of the pic (only if incorrectly used without fitting).
>  2. The hardcoded pixel size of the SVG files is omitted and now can scale to fit the container when used relying on the internal SVG styling and not the page layout.

PS: I'm not really sure if the `logo-design.svg` and `logo-on-black.svg` are useful in their current form, but as I don't know for certain if those assets are actually used anywhere, the only change made is to update the crown to the new shape (roughly translating to the aspect ratio previously used in those files).

### CODE REVIEW

**Comment by Araq** [comment]
So in order to save 23% we risk breaking existing websites that might depend on this precise layout? Why not just add logo-crown2.png, etc.

**Comment by indiscipline** [comment]
Do you ask the question because you want me to elaborate or are you just being dismissive?

* The minimal file size decrease is 77%, not 23%
* File size is not the main point of the PR
* "Breaking" existing websites is too strong of a word. For any "breakage" to happen the site need to somehow rely on the precise pixel size of the PNG file or the hardcoded pixel size of the SVG (removed with this PR).
  + PNG: The dimensions are huge, and the image is not an equal rendering of the SVG, as it omits the negative space (in other words, the transparent margins are hard-trimmed) required for proper balanced display in a layout and as a result presented in a different set of dimensions. That means the PNG is not a design-element ready for embedding, but just an asset. It's only use is to open with an image-viewing software that does not support vector graphics.
  + SVG: Hardcoding the size in pictures only makes sense if you want to recommend the exact display size of the image. Setting `width` and `size` attributes of the `<svg>` element is pre-embedding additional css styling into the svg which goes against the nature of the format (svg is _scalable_ vector graphics).
* Neither the nim-lang.org nor forum.nim-lang.org use any of the files in this repos as-is for the main logo in the left corner. The former uses an edited svg (cut from logo-design.svg) and the former uses a custom PNG. Both use the aspect ratio of the crown from `logo-design.svg` and not from the logo-crown svg/png.

In other words, currently it's all already a mess.

This PR is just a technical step of fixing the shape of he logo. If you'd like to have assets ready for professional use and direct embedding you need to change the current files anyway.

**Comment by indiscipline** [comment]
Moving forward with subsequent PRs requires answering a couple of questions:

1. Is this repo just an abstract source of truth for logos or is it a collection of assets ready for usage?
2. Which rendering of the crown is the correct one? Aspect ratio is not consistent between solo and with-label logo variations and implies this is an important distinction. If so, even the official forum is using the wrong image.

When answered, I can make a set of images ready for direct linking/embedding and a presentation image which serves as a branding guide and illustrates the proper asset usage.

**Comment by indiscipline** [comment]
If there's a fear of breaking something (I would like to know what exactly), there's an option to fully conform to the current state of the assets. I just make the PNG the same exact size (makes ~ 4 top and bottom pixel rows transparent) and add the hardcoded pixel values back to the SVGs. I feel that these concerns are based on a technicality and not a real state of the asset usage, though.

**Comment by Araq** [comment]
> Neither the nim-lang.org nor forum.nim-lang.org use any of the files in this repos as-is for the main logo in the left corner.

Ah, ok.
